spec/mongo/bulk_write_spec.rb

```diff
@@ -8,6 +8,14 @@
 
   after do
     authorized_collection.delete_many
+    collection_with_validator.drop()
+  end
+
+  let(:collection_with_validator) do
+    c = authorized_client[:validating,
+                          :validator => {:a => {'$exists' => true} }]
+    c.create
+    c
```

Comment:
Unfortunately it doesn't work when I remove the create call. I'm not 100% sure why, but it doesn't trigger the errors (I think it just treats the collection as one without a validator). Same thing for collection_spec.rb.

Maybe I am misunderstanding how collections are created with options?